forge/csrc/reportify/reportify.cpp

```diff
@@ -425,5 +458,29 @@ void dump_graph(
     std::string default_dir = get_default_reportify_path("");
     dump_graph(default_dir, test_name, graph_prefix, graph, report_path);
 }
+
+void dump_mlir(const std::string& name, mlir::ModuleOp* module)
+{
+    if (env_as<bool>("FORGE_DISABLE_REPORTIFY_DUMP"))
+        return;
+
+    std::string path = get_default_reportify_path("");
+    std::string report_path = get_mlir_reports_relative_directory();
+
```

Comment:
Let's remove the new line here to get a cleaner function blocking (e.g. first block is to build the report path).